components/Tabs/examples/MDCTabBarViewTypicalExampleViewController.m

```diff
@@ -14,13 +14,23 @@
 
 #import <UIKit/UIKit.h>
 
+#import <MaterialComponents/MaterialActionSheet+Theming.h>
+#import <MaterialComponents/MaterialActionSheet.h>
 #import <MaterialComponents/MaterialAnimationTiming.h>
 #import <MaterialComponents/MaterialContainerScheme.h>
+#import <MaterialComponents/MaterialIcons+ic_check.h>
```

Comment:
this is causing issues with the import script. it is expecting it in the format found in 
https://github.com/material-components/material-components-ios/blob/800188a64994c68d089af849ea7cd080b1a4d4ab/components/ButtonBar/examples/ButtonBarIconExample.m#L18

Reply:
@romoore can you catch me up on why <> notation was used here?

Reply:
`MDCTabBarView` is in beta, so we use the <> brackets to reference other podspecs. If this isn't necessary any longer, then I believe we can update many of our Beta component source files.

Reply:
You will probably have to update the import script to handle these Beta components until we are allowed to mark them "Ready".